app/src/main/java/org/mifos/selfserviceapp/ui/activities/ClientListActivity.java

```diff
@@ -0,0 +1,130 @@
+package org.mifos.selfserviceapp.ui.activities;
+
+import android.app.ProgressDialog;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v4.widget.SwipeRefreshLayout;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.widget.Toast;
+
+import org.mifos.selfserviceapp.R;
+import org.mifos.selfserviceapp.data.Client;
+import org.mifos.selfserviceapp.presenters.ClientListPresenter;
+import org.mifos.selfserviceapp.ui.adapters.ClientListAdapter;
+import org.mifos.selfserviceapp.ui.views.ClientListView;
+import org.mifos.selfserviceapp.utils.Constants;
+import org.mifos.selfserviceapp.utils.RecyclerItemClickListener;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.inject.Inject;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+/**
+ * @author Vishwajeet
+ * @since 14/07/16
+ */
+public class ClientListActivity extends BaseActivity implements RecyclerItemClickListener.OnItemClickListener, ClientListView {
+
+    @Inject
+    ClientListPresenter mClientListPresenter;
+
+    private ClientListAdapter clientListAdapter;
+    private List<Client> clientList = new ArrayList<>();
+    private LinearLayoutManager layoutManager;
+    private ProgressDialog progress;
+
+    @BindView(R.id.rv_clients)
+    RecyclerView rv_clients;
```

Comment:
Avoid use of snake case while naming objects in the java class and stick the camel case, this would maintain integrity in the code. Ideally you could use rvClients, (ivProfilePicture, tvFirstName) etc.